Lib/fortran/fortranfragments.swg

```diff
@@ -0,0 +1,22 @@
+/* -------------------------------------------------------------------------
```

Comment:
The python module does this in `python/pyfragments.swg` included by `python/pytypemaps.swg`, so I thought this was a capability Fortran is supposed to provide as well. (The file is included by `fortran.swg` before any of the internal fragment definitions.) I don't think any of the projects I've used with SWIG-Fortran actually depend on this capability, so it's fine to delete it,  but I'd like to double check first why it would be different from the other target languages.